handlers/network_data_aggregator.py

```diff
@@ -0,0 +1,12 @@
+import redis
```

Comment:
Should we have the copyright in the beginning of each `.py` file just like we have in the full node? @msbrogli @jansegre

Reply:
I don't _believe_ it makes too much of a difference, I think the `LICENSE.txt` file with the explicit license (we usually use Apache 2.0) is the most important (as a piece of evidence [the rustc compile](https://github.com/rust-lang/rust/blob/master/compiler/rustc_ast/src/ast.rs) is Apache2/MPL dual licensed and has absolutely no headers). However I vaguely remember we being advised to add the headers, so that's what we standardized on `hathor-core` and we try to keep it consistent. Updating the year is very annoying and I don't fully understand when we have to update or what are the implications of being outdated, so maybe that is a reason for not using headers.

Reply:
We've been using Apache 2.0 only in hathor-core. The other wallets are using MIT for now. I think it's ok and we can keep only the `LICENSE` file in the repo.